notebooks/README.md

```diff
@@ -0,0 +1,25 @@
+# Spark and Jupyter Notebooks on Dataproc
+
+Google Cloud Dataproc allows you use Jupyter Notebooks on your cluster by enabling the Jupyter component and component gateway. 
```

Comment:
Nit: please add links for "Jupyter component" and "component gateway". Component Gateway should be capitalized as well

Reply:
I have updated this